### PR TITLE
Issue 3 and Issue 11

### PR DESCRIPTION
**Pull request by GreyCoder25**
Initialise dbs directory
add scripts for creating and deleting user's database
add script for creating clients-lockers table in user's database

### CODE REVIEW

File: dbs/init_psql_db.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+sudo -u postgres psql -f ./migrations_psql/create_dbs.sql -U postgres
+sudo -u postgres psql -f ./migrations_psql/migrations_lockers.sql -U postgres -d users_db
+
+
```
Comment (AdoreIt) [inline dbs/init_psql_db.sh]: delete empty lines

File: dbs/migrations_psql/create_dbs.sql [hunk 2]
```diff
@@ -0,0 +1,4 @@
+create user cvmasters with password '1234';
+create database users_db owner cvmasters;
+
+
```
Comment (AdoreIt) [inline dbs/migrations_psql/create_dbs.sql]: delete extra empty lines

File: dbs/mongodb.py [hunk 3]
```diff
@@ -0,0 +1 @@
+import pymongo
```
Comment (AdoreIt) [inline dbs/mongodb.py]: add empty line

Comment (GreyCoder25) [reply dbs/mongodb.py]: There are empty line already

**Pull request merged**